src/pages/ExamsPage/index.scss

```diff
@@ -0,0 +1,14 @@
+@import "~@edx/paragon/scss/core/core";
+
+#exam-selector {
+  margin: 0.5em auto;
+}
+
+.pgn__searchfield .form-control {
+  border: 1px solid $search-border-color !important;
+}
+
+.pgn__searchfield.has-focus:not(.pgn__searchfield--external)::after {
```

Comment:
override some of the default styling to deal with putting searchfield inside of a select dropdown. The default focus border looks odd here and causes sizing issues so I've removed it and placed a simple border on the input element instead.